### ISSUE TITLE
link title obstructs Link preview

### ISSUE THREAD

**Issue by mliu59** [wikispeedruns/wikipedia-speedruns#294]
![image](https://user-images.githubusercontent.com/61032765/160933913-b6e9ec8d-a4c3-46f1-a4dc-b34efec03e1b.png)

**Comment by mliu59** [comment]
@maggarwal99 Maybe we can incorporate the link title into the preview itself? Like a header at the top of the box that just has the actual title of the link's page

**Comment by maggarwal99** [comment]
Yeah can def do that, Wikipedia removes the `title` attr on mouseover and puts it back on mouseout, so maybe we can do something similar.
